stories/__tests__/SQFormMultiSelect.stories.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import * as Yup from 'yup';
+import {render, screen, waitFor, within} from '@testing-library/react';
+import {composeStories} from '@storybook/testing-react';
+
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormMultiSelect.stories';
+
+const {SQFormMultiSelect} = composeStories(stories);
+
+const initialDropdownValue = '- -';
+
+afterEach(() => {});
+
+describe('Tests for SQFormMultiSelect', () => {
+  it('should render a form with a multiselect input and dropdown options', async () => {
+    render(<SQFormMultiSelect size="auto" />);
+
+    const expandButton = await waitFor(() =>
+      screen.findByRole('button', {
+        name: /friends/i
+      })
+    );
+
+    userEvent.click(expandButton);
```

Comment:
Since you're performing an action after the render, checking for the options could be moved to its own test. But I can also see how having the options is essential to the component.

Reply:
> Since you're performing an action after the render, checking for the options could be moved to its own test. But I can also see how having the options is essential to the component.

I'm going to go ahead and split this into two. For extra by-the-bookness

Reply:
done!